### ISSUE TITLE
HTML: Interactive SVG "viewHeight" is not defined

### ISSUE THREAD

**Issue by jmigual** [doxygen/doxygen#11056]
**Describe the bug**
When checking the console in Microsoft Edge I get the error:

```
ReferenceError: viewHeight is not defined
```

**Screenshots**
![image](https://github.com/user-attachments/assets/d8633544-5fb7-4c57-9296-6f5e81c21edc)

**To Reproduce**
[doxygen.tar.gz](https://github.com/user-attachments/files/16512400/doxygen.tar.gz)

**Expected behavior**
No error present in the console

**Version**
1.11.0 (9b424b03c9833626cd435af22a444888fbbb192d)
Windows 11 22H2

**Additional context**
Upon inspecting `svg.min.js`  it seems that indeed the `viewHeight` and `viewWidth` variables are not declared:

![image](https://github.com/user-attachments/assets/d9d26339-d739-46ec-b6a3-f87b54d6e2af)

**Comment by albert-github** [comment]
Looks like the problem is introduced between the versions 1.9.6 and 1.9.7 as with the 1.9.7 I get the warnings as well and with the 1.9.6 version I don't get the warnings.

**Comment by albert-github** [comment]
It looks like the problem is due to making the code more uniform and the `viewHeight` and `viewWidth` are only set in case it is needed, i.e. the zoom / pan functionality is present.
I created a small example:
```
void f1() { f2(); }
void f2() { f3(); }
void f3() { f4(); }
void f4() { f5(); }
void f5() { f6(); }
void f6() { f7(); }
void f7() { f8(); }
void f8() { f9(); }
void f9() {  }
```
and of the 16 graphs only:
```
html/test1_8cpp_a5336629efd329e940ab0935ff10da671_icgraph.svg:var viewHeight = 27;
html/test1_8cpp_a8e6c5bab59a65719e1eea10e8ea83fe9_cgraph.svg:var viewHeight = 27;
```
set the variables.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11058

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
